### DISCUSSION TITLE
How to deploy exe's using G-CLI

### DISCUSSION THREAD

**Discussion by phorn-NI** [JamesMc86/G-CLI#160] [Q&A]
Hi @JamesMc86 ,

We're in the process of deploying a project which has multiple exe's, each with a G-CLI interface. This works great when we're deploying on to a system where LV dev environment is installed, and therefore g-cli can easily be installed via VI Package Manager.

We have a build pipeline that creates a deployment file (basically a big zip) with all the exe's, automation code, plugins etc. Are we OK to pull out the g-cli proxy msi and include that in the deployment to the customer?

The other thought I has was maybe we could install it as a System package through VIPM, as its the proxy I think we need rather than any of the LV components - but that doesn't seem to be available as an option in VIPM.

![image](https://github.com/JamesMc86/G-CLI/assets/92914727/1113e614-ea3c-4777-a221-622a88ce38e5)

Is there a technical or license reason this isn't an option?

Thanks,

Pete

**Comment by JamesWiresmith** [comment]
Hi Pete,

Nope, no reason at all you can't do that and no technical reason it should cause any issues.

Cheers,
James
